### PR TITLE
Fix dilate alpha threshold & auto apply dilate on embedded texture in editor

### PR DESCRIPTION
**Pull request by Jupeyy**
editor dilate can be turned off.
about the threshold:

old:
Screenshot_20200920_175749

new:
Screenshot_20200920_175846

### CODE REVIEW

**Comment by Jupeyy** [comment]
will ask mind to test it

File: src/engine/shared/dilate.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/* (c) Magnus Auvinen. See licence.txt in the root of the distribution for more information. */
```
Comment (edg-l) [inline src/engine/shared/dilate.cpp:1]: I don't think you should include copyright with matricks here, iirc new files dont include this

Comment (Jupeyy) [reply src/engine/shared/dilate.cpp:1]: ok, i copied it from dilate, so thought i copy copyright too

**Comment by Jupeyy** [comment]
file sizes are bit bigger, bcs interpolation is cleaner

**Comment by def-** [comment]
bors r+
Seems fine to me and a good idea.

**Pull request merged**